publication-commons/src/main/java/no/unit/nva/publication/doi/dto/Contributor.java

```diff
@@ -0,0 +1,78 @@
+package no.unit.nva.publication.doi.dto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+import nva.commons.utils.JacocoGenerated;
+
+public class Contributor {
+
+    private final String id;
```

Comment:
is this an identifier or an id?

Reply:
Changed into URI, but currently there is lacking a lookup API for this. @brinxmat will schedule work for it for the next sprint.